include/netinet/in.h

```diff
@@ -217,26 +217,16 @@
 /* This macro to convert a 16/32-bit constant values quantity from host byte
  * order to network byte order.  The 16-bit version of this macro is required
  * for uIP:
- *
- *   Author Adam Dunkels <adam@dunkels.com>
- *   Copyright (c) 2001-2003, Adam Dunkels.
- *   All rights reserved.
```

Comment:
Why this is removed?

Reply:
* Unclear if we need it given we aren't using the original swaps
* It appears to conflict with the top-level copyright header (all rights reserved vs Apache)

No objection to keeping it.